.github/workflows/api-integrations-deploy.yml

```
@@ -0,0 +1,88 @@
+        options:
+          - all
+          - goldsky
+          - defillama
+          - coingecko
+
+jobs:
+  deploy-integrations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)